src/tests/fs_tests.rs

```diff
@@ -17,7 +17,9 @@ mod fs_tests {
         // ut_lind_fs_fdflags();
         // ut_lind_fs_file_link_unlink();
         // ut_lind_fs_file_lseek_past_end();
-        ut_lind_fs_fstat_complex();
+        // ut_lind_fs_fstat_complex();
+        // ut_lind_fs_rmdir();
```

Comment:
Why are these commented out?

Reply:
All the tests except the latest added one are commented out. I guess people want to focus on the latest one only. Should I uncomment all the tests or it's okay to have them this way since we are still adding more tests?

Reply:
Hi all -- that is my bad. I wrote the rmdir test before your PR was merged, so I commented it out. Feel free to comment them back in -- my current PR does the same thing and will update main when it is merged.